deno/config.ts

```diff
@@ -11,6 +11,14 @@ try {
   Deno.exit(exitCodes.ImportError)
 }
 
+if (func.default === undefined) {
```

Comment:
I understand that this level of granularity gives people more information — i.e. no export vs. export with wrong type. But do you think it's worth the additional logic for the extra branch? If we just do the `typeof` check and show the same message for the two cases with a `ntl.fyi` link pointing to https://docs.netlify.com/edge-functions/api/, I think that would achieve the same goal with half the logic.

What do you think?

Reply:
I think it makes sense, I'll implement it.

Reply:
This is now done _(waiting on a `ntl.fyi` link though)_.

Reply:
now done done